### ISSUE TITLE
Default values accumulated in help

### ISSUE THREAD

**Issue by thorstenkampe** [jeffrimko/Qprompt#4]
```
>>> qprompt.ask_int(dft = 1)
[?] Enter an integer [1]: ?
[1, <class 'int'>]
[?] Enter an integer [1]: 1
1
>>> qprompt.ask_int(dft = 2)
[?] Enter an integer [2]: ?
[1, 2, <class 'int'>]
[?] Enter an integer [2]: 33
33
>>> qprompt.ask_int(dft = 44)
[?] Enter an integer [44]: ?
[1, 2, 44, <class 'int'>]
[?] Enter an integer [44]:
```

**Comment by jeffrimko** [comment]
Weird! Good catch. Will investigate.

**Comment by jeffrimko** [comment]
Should be fixed in `0.9.4`.
